jsonnet/kube-prometheus/prometheus/prometheus.libsonnet

```diff
@@ -440,5 +440,37 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           ],
         },
       },
+    serviceMonitorAwsEksCNI:
```

Comment:
this should not be in the default, only in the eks patch, like the service

Reply:
cbbfa0cad5ca7e231f8a255e95255d75fb75612c moves the service monitor to patch.